packages/scene-composer/src/components/panels/scene-components/AnchorComponentEditor.tsx

```diff
@@ -118,23 +123,25 @@ export const AnchorComponentEditor: React.FC<IAnchorComponentEditorProps> = ({
   );
 
   const iconOptions = useMemo(() => {
-    return Object.keys(SCENE_ICONS).map((sceneIcon) => ({
-      label: intl.formatMessage(i18nSceneIconsKeysStrings[sceneIcon]) || sceneIcon,
-      value: sceneIcon,
-    }));
+    return Object.keys(tagStyle ? SCENE_ICONS_CUSTOM : SCENE_ICONS).map((sceneIcon) => {
```

Comment:
shouldn't need to use different enums here, just don't add or remove Custom when flag is off